easybuild/easyblocks/q/qscintilla.py

```diff
@@ -87,31 +107,33 @@ def install_step(self):
                 raise EasyBuildError("Failed to change to %s: %s", pydir, err)
 
             # apparently this directory has to be there
-            qsci_sipdir = os.path.join(self.installdir, 'share', 'sip', 'PyQt4')
+            qsci_sipdir = os.path.join(self.installdir, 'share', 'sip', self.pyqt_pkg_name)
             mkdir(qsci_sipdir, parents=True)
 
-            pylibdir = os.path.join(det_pylibdir(), 'PyQt4')
-
-            pyqt = get_software_root('PyQt')
-            if pyqt is None:
-                raise EasyBuildError("Failed to determine PyQt installation prefix, PyQt not included as dependency?")
+            pylibdir = os.path.join(det_pylibdir(), self.pyqt_pkg_name)
 
             cfgopts = [
                 '--destdir %s' % os.path.join(self.installdir, pylibdir),
                 '--qsci-sipdir %s' % qsci_sipdir,
                 '--qsci-incdir %s' % os.path.join(self.installdir, 'include'),
                 '--qsci-libdir %s' % os.path.join(self.installdir, 'lib'),
-                '--pyqt-sipdir %s' % os.path.join(pyqt, 'share', 'sip', 'PyQt4'),
+                '--pyqt-sipdir %s' % os.path.join(self.pyqt_root, 'share', 'sip', self.pyqt_pkg_name),
                 '--apidir %s' % os.path.join(self.installdir, 'qsci', 'api', 'python'),
                 '--no-stubs',
+                '--no-dist-info',
```

Comment:
@Darkless012 It looks like this `--no-dist-info` flag is only supported in recent version of `QScintilla`, certainly not for version 2.10:

```
configure.py: error: no such option: --no-dist-info
```

Reply:
It was added in 2.10.7, see https://www.riverbankcomputing.com/news/qscintilla-2107, so this needs a version check...

Reply:
fixed in 5de67dc